requests/torchcodec.yaml

```
@@ -0,0 +1,3 @@
+action: add_feedstock_output
+feedstock_to_output_mapping:
+  - torchcodec: "torchcodec-*"
```

```suggestion
  - torchcodec: "torchcodec-tests"
```

I personally don't like globbing. is this acceptable?

---

Sure! That was indeed mentioned in the original post, even if there were a few copy&paste error for which the message was confusing.